test/models/test_model_factory.py

```diff
@@ -19,7 +19,6 @@
 
 parametrize = pytest.mark.parametrize('model', [
     pytest.param(ModelType.GPT_3_5_TURBO, marks=pytest.mark.model_backend),
-    pytest.param(ModelType.GPT_3_5_TURBO_16K, marks=pytest.mark.model_backend),
     pytest.param(ModelType.GPT_4, marks=pytest.mark.model_backend),
```

Comment:
How about changing all GPT4 in tests to GPT_4_TURBO? First, the GPT4 will point to GPT_4_TURBO soon. Second, GPT4 is more expensive than GPT4_TURBO. @lightaime

Reply:
Sounds good!

Reply:
done